src/v2/view-builder/views/captcha/CaptchaView.js

```diff
@@ -74,7 +135,11 @@ export default View.extend({
 
     // Attaching the callback to the window object so that the CAPTCHA script that we dynamically render
     // can have access to it since it won't have access to this view's scope.
-    window.OktaSignInWidgetOnCaptchaLoaded = onCaptchaLoaded;
+    window.OktaSignInWidgetOnCaptchaLoaded = this.onCaptchaLoaded;
```

Comment:
do you have to bind to `this` context as you can access the function under `window` object in test?

Reply:
Yeah this is a good point! I don't need to bind to `this` because it was mostly done to facilitate unit testing.